apps/dash-datashader/app.py

```diff
@@ -0,0 +1,316 @@
+# coding: utf-8
+
+import dash
```

Comment:
We should try to reorder the imports to be PEP-8 compliant, as black does not take care of that. Please see: https://www.python.org/dev/peps/pep-0008/#imports